src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: 300ms;
```

Comment:
```suggestion
    transition: 300ms;
```
 transition: color 300ms || transition: color $duration

Reply:
I agreee with @miftaha, it is considered to be a better practice to write the transition property explicitly